test/service.go

```diff
@@ -108,6 +108,35 @@ func CheckDeploymentScale(ctx *Context, ns, name string, scale int) error {
 	return nil
 }
 
+func RouteHost(ctx *Context, ns, name string) (string, error) {
+	r, err := ctx.Clients.Route.Routes(ns).Get(name, metav1.GetOptions{})
+	if err != nil {
+		return "", err
+	}
+	ready, _ := RouteHasHost(r, nil)
+	if !ready {
+		return "", fmt.Errorf("route %s/%s is not ready yet")
+	}
+	return r.Status.Ingress[0].Host, nil
+}
+
+func VerifyConcoleCLIDownload(ctx *Context, name, host string) error {
```

Comment:
Why not inline this check? I don't see us reusing it.

Reply:
inlined.